tests/Command/PullRequest/PullRequestMergeCommandTest.php

```diff
@@ -67,26 +67,14 @@ class PullRequestMergeCommandTest extends BaseTestCase
 OET;
 
     const COMMAND_DISPLAY = <<<OET
-! [NOTE] Adding remote "gushphp" with "https://github.com/gushphp/gush.git".
```

Comment:
This done by the GitConfigHelper :) and since thats mocked it doesn't output anything anymore.